src/RBDyn/Coriolis.h

```diff
@@ -0,0 +1,84 @@
+#ifndef CORIOLIS_H_KED6CAHK
+#define CORIOLIS_H_KED6CAHK
+
+#include <array>
+
+#include <RBDyn/MultiBodyConfig.h>
+#include <RBDyn/CoM.h>
+#include <rbdyn/config.hh>
+
+namespace rbd
+{
+
+using Block = std::array<int, 3>;
+using Blocks = std::vector<Block>;
+
+/** Expand a symmetric product of a jacobian by its transpose onto every DoF.
```

Comment:
Well `jacMat` can be anything, but it *must* be a `dof x dof` matrix. Then, it will be expanded by blocks of `dof_j1 x dof_j2` which means that a `jacMat` not representing a `J^T J`  (symmetric product) is meaningless.